### ISSUE TITLE
Add configuration to disable rules

### ISSUE THREAD

**Issue by jnmoore-intel** [tlvince/eslint-plugin-jasmine#91]
I'm getting eslint warnings for jasmine/no-suite-dupes when I have suites that use variables in their name.

this is pretty rudimentary. In my code, I have a lot more `it` suites and more logic, but that gets in the way of the example:

```
let queue= [true, true, false],
    shouldReturnFalse = (state) => {
    describe(`from ${state}`), () => {
        it('should return false', () => {
            expect(result).toBe(false)
        })
    }),
    shouldReturnTrue = (state) => {
    describe(`from ${state}`), () => {
        it('should return true', () => {
            expect(result).toBe(true)
        })
    })

describe('object', () => {
    let result
    beforeEach(() => {
        result = queue.pop()
    }
    shouldReturnTrue(1)
    shouldReturnTrue(2)
    shouldReturnFalse(3)
})

```

Running eslint on the spec generates this warning:

```
  112:17  warning  Duplicate suite: "from null"                             jasmine/no-suite-dupes
```

Like other eslint rules, I would like to configure eslint-plugin-jasmine to ignore `jasmine/no-suite-dupes`, but the [documentation](https://github.com/tlvince/eslint-plugin-jasmine/blob/master/docs/rules/no-suite-dupes.md) does not provide this mechanism

**Comment by jnmoore-intel** [comment]
The functionality is there, and I'm able to turn off the rule with `"jasmine/no-spec-dupes": "off"` entry in my .eslintrc.json file, but the documentation should include an example

**Comment by burabure** [comment]
Hey @jnmoore-intel =)
 Turning rules on or off is part of the basic eslint features: you can learn more about it here http://eslint.org/docs/user-guide/configuring#configuring-rules
